### DISCUSSION TITLE
无法输入中文，输入中文后崩溃

### DISCUSSION THREAD

**Discussion by renyibing** [CefView/QCefView#142] [Q&A]
无法输入中文，输入中文后崩溃

**Comment by L-Super** [comment]
加一

**Reply by tishion** [reply]
搜狗输入法？

**Reply by L-Super** [reply]
Windows 11原生输入法

**Reply by tishion** [reply]
CEF和Qt版本提供下，我一直都是Windows 11，没遇到过这种情况

**Reply by L-Super** [reply]
- cef_binary_111.2.7+gebf5d6a+chromium-111.0.5563.148_windows64

- Qt 5.15.2
106以及目前最新112都测试过，现象一致。具体详情可参考[#238](https://github.com/CefView/QCefView/issues/238)
